pkg/tweetnacl/Makefile

```diff
@@ -0,0 +1,11 @@
+PKG_NAME=tweetnacl
+PKG_URL=https://github.com/kYc0o/tweetnacl
+PKG_VERSION=3548e2c0474731123e8e22a8049a945cd227aea6
+
+.PHONY: all
+
+all: git-download
+	@cp Makefile.tweetnacl $(PKG_BUILDDIR)/Makefile
```

Comment:
Do you mean adding such Makefile to the repo? I wanted to create the repo as generic as possible...